utils/testutils/bloomd_addrs.go

```diff
@@ -0,0 +1,62 @@
+package testutils
+
+import (
+	"fmt"
+	"net/url"
+	"os"
+	"strings"
+	"sync"
+	"testing"
+)
+
+var BloomdAddrs = addrsFromEnv
+
+func UseAddrsFromEnv() {
```

Comment:
is it used?

Reply:
No. I was thinking to move a bloomd mock as well. I'll delete it for now